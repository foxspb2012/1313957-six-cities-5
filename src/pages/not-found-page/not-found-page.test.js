import React from 'react';
import history from '../../history';
import {Router} from 'react-router-dom';
import renderer from 'react-test-renderer';
import NotFoundPage from './not-found-page';

it(`Should NotFoundPage render correctly`, () => {
  const tree = renderer
    .create(
        <Router history={history}>
          <NotFoundPage />
        </Router>

    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
