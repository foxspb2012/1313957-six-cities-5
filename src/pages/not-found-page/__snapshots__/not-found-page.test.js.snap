// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should NotFoundPage render correctly 1`] = `
<main
  className="page__main page__main--index page__main--index-empty"
>
  <div
    className="cities"
  >
    <div
      className="cities__places-container cities__places-container--empty container"
    >
      <section
        className="cities__no-places"
      >
        <div
          className="cities__status-wrapper tabs__content"
        >
          <b
            className="cities__status"
          >
            Page not found
          </b>
          <p
            className="cities__status-description"
          >
            Click button to visit the main page
          </p>
          <a
            className="button form__submit"
            href="/"
            onClick={[Function]}
          >
            Go to the main page
          </a>
        </div>
      </section>
      <div
        className="cities__right-section"
      />
    </div>
  </div>
</main>
`;
